src/main/java/soconnect/model/util/SampleDataUtil.java

```diff
@@ -42,6 +45,17 @@ public static Person[] getSamplePersons() {
         };
     }
 
+    public static Todo[] getSampleTodos() {
+        return new Todo[] {
+            new Todo(new Description("Watch recorded lecture videos"), new Priority("high"),
+                getTagSet("friends")),
+            new Todo(new Description("Revise content"), new Priority("medium"),
+                getTagSet("family")),
+            new Todo(new Description("Prepare for tutorials"), new Priority("low"),
+                getTagSet("friends", "family"))
+        };
+    }
+
```

Comment:
```suggestion
    public static Todo[] getSampleTodos() {
        return new Todo[] {
            new Todo(new Description("Watch recorded lecture videos"), new Priority("high"),
                getTagSet("math")),
            new Todo(new Description("Revise content"), new Priority("medium"),
                getTagSet("english")),
            new Todo(new Description("Prepare for tutorials"), new Priority("low"),
                getTagSet("math", "chinese"))
        };
    }

```

Hmmm the description does not really match with the tag. Can include one more `Todo` without any tag.